spec/gameSpec.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+describe('Game', function() {
+  let game
+
+  beforeEach(function() {
+    game = new Game();
+    game.beginFrame();
+  })
+
+  it('begins a new frame',function() {
+    expect(game.turn).toEqual(1)
+  })
+
+  it('has an empty scorecard at first', function() {
+    expect(game.scorecard).toEqual([])
+  })
+
+  it('adds the users points to each respective frame', function() {
+    game.throwBall(7);
+    game.addToScorecard();
+    expect(game.scorecard[0]).toEqual([ 7 ])
+  })
+
+  it('calculates the total score', function() {
+    game.throwBall(7);
+    game.throwBall(2);
+    game.addToScorecard();
+    game.throwBall(5);
+    game.throwBall(4);
+    game.addToScorecard();
+    expect(game.totalPoints()).toEqual(18)
+    
+  })
+
+  describe('when the player throws a strike', function() {
+
+    beforeEach(function() {
+      game.throwBall(10);
+      game.addToScorecard();
+      game.beginFrame();
+      game.throwBall(7)
+      game.throwBall(2);
+      game.addToScorecard();
+      game.strikeBonus();
+    })
+
+    it('adds a strike bonus', function() {
+      expect(game.scorecard[0]).toEqual([10, 7, 2])
```

Comment:
Similarly, what is the user-observable effect of adding a strike bonus? Can this expectation be expressed in terms of `game.totalPoints()`?